src/AzSK.ADO/Framework/Core/AzSKADOInfo/OrganizationInfo.ps1

```diff
@@ -0,0 +1,52 @@
+using namespace System.Management.Automation
+Set-StrictMode -Version Latest
+
+class OrganizationInfo: CommandBase {
+
+    hidden [string] $organizationName;
+    hidden [PSObject[]] $projects = @();
+
+    OrganizationInfo([string] $organizationName, [PSObject[]] $projects, [InvocationInfo] $invocationContext):
+    Base($organizationName, $invocationContext) {
+        $this.organizationName = $organizationName;
+        $this.projects = $projects;
```

Comment:
should we make it +=
as lot of times we have a conflict of singleton array vs list